### PR TITLE
Javascript/javascript2/week3

### PR DESCRIPTION
**Pull request by islam-fawzy25**

### CODE REVIEW

File: javascript/javascript2/week3/warmUp/warmUp.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+runAfterDelay(4,  () => console.log('Should be logged after 4 seconds'));
+//8./////////////////////////////////////////////////////////////////////
+
+const body = document.querySelector('body');
+
+body.addEventListener('dblclick', function () {
+    setTimeout(() => {
+        console.log('double click!');
+    }, 500);
+});
```
Comment (cecastosic) [inline javascript/javascript2/week3/warmUp/warmUp.js:82]: The Idea is to log out the text: "double click!",  If a double click has been detected, no need for setTimeout

```suggestion
body.addEventListener("dblclick", function () {
  console.log("double click!");
});

```

File: javascript/javascript2/week3/classEx.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+/*
```
Comment (cecastosic) [inline javascript/javascript2/week3/classEx.js:1]: Is this file part of homework? If not, I would remove it from PR

File: javascript/javascript2/week3/warmUp/warmUp.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```
Comment (cecastosic) [inline javascript/javascript2/week3/warmUp/warmUp.html:1]: format file with Prettier

File: javascript/javascript2/week3/fasterPresser/index.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
+body {
+    background-color: cornflowerblue;
+    text-align: center;
+}
+
+#countS {
+    float: left;
```
Comment (cecastosic) [inline javascript/javascript2/week3/fasterPresser/index.css:7]: maybe flex is better solution, then using float left and right

**Review by cecastosic — COMMENTED** [review]
Great job! Some comments: 

- Use Prettier to format your code, 
- try to use em or rem, instead of px, and 
- flex instead of float left and right
- remove commented code that is not part of homework
- be careful with names, for both ids and variables, all should be descriptive (btn1, btn2 not the best choice, constFunction or functionAsConts)

File: javascript/javascript2/week3/warmUp/warmUp.html (hunk 3 above)
Comment (islam-fawzy25) [reply javascript/javascript2/week3/warmUp/warmUp.html:1]: > format file with Prettier

sorry, but what dose it mean  that ?

File: javascript/javascript2/week3/fasterPresser/index.css (hunk 4 above)
Comment (islam-fawzy25) [reply javascript/javascript2/week3/fasterPresser/index.css:7]: > maybe flex is better solution, then using float left and right

you mean     display: flex; ?
sorry i am not so good with css

File: javascript/javascript2/week3/warmUp/warmUp.html (hunk 3 above)
Comment (cecastosic) [reply javascript/javascript2/week3/warmUp/warmUp.html:1]: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode and format files with it

File: javascript/javascript2/week3/fasterPresser/index.css (hunk 4 above)
Comment (cecastosic) [reply javascript/javascript2/week3/fasterPresser/index.css:7]: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Comment (islam-fawzy25) [reply javascript/javascript2/week3/fasterPresser/index.css]: > https://css-tricks.com/snippets/css/a-guide-to-flexbox/

yes , Thanks for helping

**Pull request merged**